packages/puppeteer-core/src/bidi/core/Request.ts

```diff
@@ -196,7 +197,7 @@ export class Request extends EventEmitter<{
 
   get hasPostData(): boolean {
     // @ts-expect-error non-standard attribute.
```

Comment:
Can we remove this `@ts-expect-error` or at least update the comment on why we need it.